package.json

```diff
@@ -54,22 +54,23 @@
   ],
   "dependencies": {
     "immutable-json-patch": "^5.1.3",
+    "parse-address": "^1.1.2",
     "seedrandom": "^3.0.5",
-    "sjcl": "^1.0.8",
-    "parse-address": "^1.1.2"
```

Comment:
I think we can revert this and package-lock.json as there should be no changes now.

Reply:
Current changes only update some dev dependencies and sort entries in the package.json. This seems harmless, let's keep it?